llvmlite/binding/context.py

```diff
@@ -1,12 +1,18 @@
 from llvmlite.binding import ffi
 
+# FIXME: Remove me once typed pointers are no longer supported.
+from llvmlite import _disable_opaque_pointers
+from ctypes import c_bool
+
 
 def create_context():
-    return ContextRef(ffi.lib.LLVMPY_ContextCreate())
+    return ContextRef(
+        ffi.lib.LLVMPY_ContextCreate(not _disable_opaque_pointers))
```

Comment:
In line with the changed name / semantics mentioned above (needs applying throughout the uses within this file):

```suggestion
        ffi.lib.LLVMPY_ContextCreate(opaque_pointers_enabled))
```